resource-managers/kubernetes/integration-tests/kerberos-yml/kerberos-set.yml

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: kerberos
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: hdfs-kerberos
+      kerberosService: kerberos
+      job: kerberostest
+  template:
+    metadata:
+      annotations:
+        pod.beta.kubernetes.io/hostname: kerberos
+      labels:
+        name: hdfs-kerberos
+        kerberosService: kerberos
+        job: kerberostest
+    spec:
+      containers:
+      - command: ["sh"]
+        args: ["/start-kdc.sh"]
+        name: kerberos
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - mountPath: /var/keytabs
+          name: kerb-keytab
+      restartPolicy: Always
+      volumes:
+      - name: kerb-keytab
+        persistentVolumeClaim:
```

Comment:
With a `StatefulSet`, you don't need to explicitly manage PVCs. You can use `.spec.persistentVolumeClaimTemplate`. The StatefulSet controller automatically creates the PV (or binds to the existing one it created before).

Reply:
+1